libs/entry-components/styles/modules/_default-theme.scss

```diff
@@ -66,25 +66,27 @@ $theme: (
 		),
 		cells: (
 			edge-gap: 4px,
-			padding: null
+			padding: null,
 		),
 		rows: (
+			odd-even-background: (
+				odd: #F0F0F0, 
+				even: null,
```

Comment:
This shouldn't be null but whitish color from AM (again, not to have breaking changes).